app/styles/modules/_input-row.scss

```diff
@@ -0,0 +1,101 @@
+html[dir='ltr'] {
+  input[type='email'],
+  input[type='text'],
+  input[type='password'] {
+      @include formElement();
+      padding: 0 0 0 $input-left-right-padding;
+  }
+}
+
+html[dir='rtl'] {
+  input[type='email'],
+  input[type='text'],
+  input[type='password'] {
+      @include formElement();
+      padding: 0  $input-left-right-padding 0 0;
+  }
+}
+
+.input-row {
+  margin-bottom: 12px;
+  position: relative;
+  width: 100%;
+
+  &:hover input {
+    border-color: $input-border-color;
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  button,
+  input {
+    @include font();
+  }
+
+  input[type='email'],
+  input[type='text'] {
+  }
```

Comment:
dude, this has been SHIPPING believe it or not check out https://github.com/mozilla/fxa-content-server/blob/master/app/styles/_general.scss#L382